### DISCUSSION TITLE
Templating & content for all types

### DISCUSSION THREAD

**Discussion by mcsdodo** [house-of-abbey/GarminHomeAssistant#148] [Ideas]
Hello! Great job with the app! Would it be possible to add templating possibilities for all types? Let's say I have a group, there are multiple lights in it. On the top view (when navigating items) I would like to have the "content" so I can check my custom state.

**Comment by josephabbey** [comment]
This is an interesting idea, certainly within the realms of possibility. Can you elevate this to a feature request issue please?

**Reply by philipabbey** [reply]
Closing this discussion as the feature had been implemented.
